File: 3048 - Sequência Secreta.cs
using System;
class Script
{
    static private void Main()
    {

        int length = Convert.ToInt32(Console.ReadLine());
        int corrects = 0;
        int lastnumber = 0;
        for (int i = 0; i < length; i++)
        {
            int number = Convert.ToInt32(Console.ReadLine());
            if (number != lastnumber)
            {
                corrects++;
            }
            lastnumber = number;
        }
        Console.WriteLine(corrects);
    }
}

File: 3048 - Sequência Secreta.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let length = Number(lines[0])
let corrects = 0;
let lastnumber = 0;
for (i = 0; i < length; i++){
    number = lines[i+1]
    if (number != lastnumber){
                corrects++;
            }
            lastnumber = number;
        }
console.log(corrects);
